src/actors/worker.rs

```diff
@@ -193,6 +194,7 @@ impl WorkerActor {
             supervisor: supervisor,
             logger: logger,
             repo: repo,
+            tests_since_gc: 0,
```

Comment:
And then this `try_run` function will accept a `git_gc_threshold` parameter that is used to initialize the `WorkerActor`'s member of the same name.

We pass that parameter through `Worker::spawn`, which is called by `SupervisorActor::spawn_workers`, which has access to `self.opts`, which is an `Options` instance defined in `src/lib.rs`. We can add a member to `Options` for the git gc threshold. To hook this up to the command line flags, crib from the code in `src/bin/preduce.rs`, which parses command line flags and creates an `Options` object from them.

Let me know if you have questions about this!